osschallenge/tests/test_selenium_no_users_in_ranking.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from selenium import webdriver
+from osschallenge.tests.pages.ranking import RankingPage
+from django.test import Client
+
+
+class MydriverTests(StaticLiveServerTestCase):
+
+    @classmethod
+    def setUpClass(self):
+        super(MydriverTests, self).setUpClass()
+        options = webdriver.ChromeOptions()
+        options.add_argument('headless')
+        options.add_argument('window-size=1200x600')
```

Comment:
Do you have to repeat these options in all the selenium tests? If yes, then I'd try to refactor that.

Reply:
I have refactored these lines of code.